packages/odyssey-react-mui/src/Tile.tsx

```diff
@@ -0,0 +1,147 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { ReactElement, memo, useMemo } from "react";
+
+import { NullElement } from "./NullElement";
+import {
+  Card as MuiCard,
+  CardActions as MuiCardActions,
+  CardActionArea as MuiCardActionArea,
+} from "@mui/material";
+import { Button } from "./Button";
+import { ButtonContext } from "./ButtonContext";
+import { Heading5, Paragraph, Support } from "./Typography";
+import { MoreIcon } from "./icons.generated";
+import { HtmlProps } from "./HtmlProps";
+import styled from "@emotion/styled";
+import {
+  DesignTokens,
+  useOdysseyDesignTokens,
+} from "./OdysseyDesignTokensContext";
+import { MenuButton } from "./MenuButton";
+
+export type TileProps = {
+  description?: string;
+  image?: ReactElement | NullElement; // Icon or image
+  menuItems?: ReactElement;
+  overline?: string;
+  title?: string;
+} & ( // You can't have actions and onClick at the same time
+  | {
+      onClick: () => void;
+      button?: never;
+      menuItems?: never;
+    }
+  | {
+      onClick?: never;
+      button?: ReactElement<typeof Button> | NullElement;
+      menuItems?: ReactElement;
+    }
+) &
+  HtmlProps;
+
+const ImageContainer = styled.div<{
+  odysseyDesignTokens: DesignTokens;
+  hasMenuItems: boolean;
+}>`
+  display: flex;
+  align-items: flex-start;
+  max-height: 64px;
+  margin-block-end: ${(props) => props.odysseyDesignTokens.Spacing5};
+  padding-right: ${(props) =>
+    props.hasMenuItems ? props.odysseyDesignTokens.Spacing5 : 0};
```

Comment:
Where's this `max-height` value coming from? Is it possible to pull it in from design tokens?

Reply:
For context: We had some discussion about this value being pixel based rather than rem based. We decided px makes the most sense here. And, we don't have px values in tokens currently

Reply:
We don't have (or currently need) a spacing token for 64px — afaik, this is the only place we use it. It's specific to the height of the Tile image, and it deliberately doesn't scale with the text (ie, it should always be max of 64px, even if the font is zoomed)